annual_dual_failure_probability.py

```
@@ -71,41 +87,39 @@ def calculate_system_mtbf(component_mttfs: dict) -> float:
     return 1.0 / inverse_sum
 
 
-def estimate_annual_dual_failure_probability(system_mtbf: int, rto_hours: int):
+def estimate_annual_dual_failure_probability(system_mtbf: int, rto_hours: int) -> tuple[float, float]:
```

Ruff (0.12.2)

Missing argument description in the docstring for `estimate_annual_dual_failure_probability`: `system_mtbf`

(D417)